tests/foreman/cli/test_hostgroup.py

```diff
@@ -3,7 +3,15 @@
 
 from fauxfactory import gen_string
 from robottelo.cli.hostgroup import HostGroup
-from robottelo.cli.factory import make_hostgroup
+from robottelo.cli.proxy import Proxy
+from robottelo.cli.factory import (
+    CLIFactoryError,
+    make_environment,
+    make_hostgroup,
+    make_location,
+    make_org,
+    make_os
```

Comment:
Nitpick: add a trailing comma. Doing so makes it easier to add, rearrange and remove items without mucking with commas. Also, adding a trailing comma may make future diffs more concise.